charts/kubeclarity/templates/deployment.yaml

```diff
@@ -48,6 +48,7 @@ spec:
               cpu: "10m"
             limits:
               memory: "100Mi"
+              cpu: "100m"
```

Comment:
I can imagine such an app to spike resources at moments. Would these limits then be enough?

Reply:
since it's a simple await db readiness job the resource requirements are very small, this is the default value declared in the the chart and has not caused any issues